stdlib/public/SDK/Foundation/Data.swift

```diff
@@ -664,6 +678,8 @@ internal final class _DataStorage {
             _backing = .customMutableReference(data)
         case .customMutableReference(let d):
             d.replaceBytes(in: range, withBytes: replacementBytes, length: replacementLength)
+        @unknown default:
+            fatalError("Unknown Data backing type")
```

Comment:
You're going to have to come back and do something about these. If Data really can grow new representations, you'll need to make sure they're handled even in inlined code. Sadly we already have a pattern for this: https://twitter.com/UINT_MIN/status/1020503397357760512

Reply:
Argh :( ok, I should talk to @phausler about this

Reply:
this code is going to have some changes coming in soon anyhow.